examples/meta/src/lib.rs

```diff
@@ -55,8 +89,41 @@ meta! {
     extra: Id, Person, Wallet
 }
 
+static mut WALLETS: Vec<Wallet> = Vec::new();
+
 #[no_mangle]
-pub unsafe extern "C" fn handle() {}
+pub unsafe extern "C" fn handle() {
+    let message_in: MessageIn = msg::load().unwrap();
+    let message_out = MessageOut::from(message_in);
```

Comment:
The gold rule says "implement `From`, use `Into`". The reason is that `Into` has a blanket implementation for
any `T` that implements `From`, so regardless of whether a type explicitly implements `From` or `Into`, it implements `Into`.
```suggestion
    let message_out: MessageOut = message_in.into();
```